ws-nextjs-app/pages/[service]/[[...]].page.tsx

```diff
@@ -68,3 +90,22 @@ export const getServerSideProps: GetServerSideProps = async context => {
     },
   };
 };
+
+export default function PageTypeToRender({ pageType, ...rest }: PageProps) {
```

Comment:
This is a bit easier to understand
```suggestion
export default function PageTypeToRender({ pageType, ...props }: PageProps) {
```